src/components/UserItem.js

```diff
@@ -2,35 +2,31 @@ import React, { Component } from 'react'
 import { css } from 'emotion'
 import styled from 'react-emotion'
 
-const Wrapper = styled('section')`
+const Wrapper = styled('div')`
   display: flex;
 `
 
 export default class UserItem extends Component {
   render() {
     return (
-      <div>
-        {this.props.users.map(user => (
-          <Wrapper>
-            <div>
-              <img src={user.photo} alt="" width="100px" />
-            </div>
-            <div
-              className={css`
-                margin-left: 5px;
-              `}
-            >
-              <h3>
-                {user.name} | {user.location}
-              </h3>
-              <p>{user.status}</p>
-              <p>
-                {user.skills[0]} {user.skills[1]} {user.skills[2]}
-              </p>
-            </div>
-          </Wrapper>
-        ))}
-      </div>
+      <Wrapper>
+        <div>
+          <img src={this.props.photo} alt="" width="100px" />
+        </div>
+        <div
+          className={css`
+            margin-left: 5px;
+          `}
+        >
+          <h3>
+            {this.props.name} | {this.props.location}
+          </h3>
+          <p>{this.props.status}</p>
+          <p>
+            {this.props.skills[0]} {this.props.skills[1]} {this.props.skills[2]}
```

Comment:
Probier mal bitte so etwas: `{this.props.skills.join(' ')}`. Geht das?

Reply:
Funktioniert, wird aber eventuell unübersichtlich sobald mehr skills im state hinterlegt werden. Lasse es aber jetzt erstmal so.